### ISSUE TITLE
Only training limited images out of a dataset

### ISSUE THREAD

**Issue by j3nsykes** [carpedm20/DCGAN-tensorflow#252]
I have a dataset of 500 images - I realise this is still quite a small dataset. However, the main issue I need to overcome is whenever I train this dataset it is only ever training 8 images at a time. See attached screenshot. This is my own dataset but the images are all the same dimension, crop and colour profile.

Am I missing something obvious I haven't executed or a default I need to change in the utils.py?

many thanks
<img width="511" alt="screen shot 2018-01-07 at 18 15 31" src="https://user-images.githubusercontent.com/31961310/34652512-115ea694-f3d7-11e7-9140-7600e2913332.png">

**Comment by neelindap** [comment]
Hi,

The 8 is the no. of batches. This model utilizes batch normalization and guessing you haven't changed the default batch size of 64, it is dividing the 500 images into batches of 64 images(and hence the 8)
You can try changing the batch size in main.py file if you want. The code will throw an error though if the number is not a perfect square.

**Comment by j3nsykes** [comment]
Thanks for the reply that makes a lot of sense now.
